internal.h

```diff
@@ -2035,7 +2035,7 @@ rb_pid_t rb_fork_async_signal_safe(int *status, int (*chfunc)(void*, char *, siz
 VALUE rb_execarg_new(int argc, const VALUE *argv, int accept_shell, int allow_exc_opt);
 struct rb_execarg *rb_execarg_get(VALUE execarg_obj); /* dangerous.  needs GC guard. */
 VALUE rb_execarg_init(int argc, const VALUE *argv, int accept_shell, VALUE execarg_obj, int allow_exc_opt);
-int rb_execarg_addopt(VALUE execarg_obj, VALUE key, VALUE val);
+int rb_execarg_adopt(VALUE execarg_obj, VALUE key, VALUE val);
```

Comment:
Is it true that this change might break a gem with native extensions that calls `rb_execarg_addopt`?

Reply:
@mikegee 
Thanks. As I mentioned in the description of this PR, I am willing to revert the change if it is suspicious. I will do if the team is not 100% confident about the safety.

>Also, if the change on the method rb_execarg_addopt should not be included, I can revert the change (I judged the change is safe enough, though).

Reply:
I totally read your description, I swear. 😃 

I have no idea if this is safe or not. I'm glad you investigated it 👍

Reply:
It stands for "add option", not single word "adopt", I think.

Reply:
I now understand that the name is not a typo. Thank you 🙇